diff --git a/node_modules/jsnes/package.json b/node_modules/jsnes/package.json
index 0000000..0000000 100644
--- a/node_modules/jsnes/package.json
+++ b/node_modules/jsnes/package.json
@@ -11,7 +11,8 @@
       "import": "./src/index.js",
       "require": "./dist/jsnes.js",
       "default": "./src/index.js"
-    }
+    },
+    "./*": "./*"
   },
   "repository": {
     "type": "git",
diff --git a/node_modules/jsnes/src/ppu/index.js b/node_modules/jsnes/src/ppu/index.js
index 0000000..0000000 100644
--- a/node_modules/jsnes/src/ppu/index.js
+++ b/node_modules/jsnes/src/ppu/index.js
@@ -1730,7 +1730,7 @@ export default class PPU {
           // top tile is (index & $FE), bottom tile is (index & $FE) + 1.
           let sprBaseAddr = (sprTile & 1) !== 0 ? 0x1000 : 0x0000;
           let topTileNum = sprTile & 0xfe;
-          let top = (sprTile & 1) !== 0 ? topTileNum - 1 + 256 : topTileNum;
+          let top = (sprTile & 1) !== 0 ? topTileNum + 256 : topTileNum;

           let dy = sprY + 1;
           let fineY = scan - dy;
